### PR TITLE
Set course progress reminder email days in configuration file.

### PR DESCRIPTION
**Pull request by Faizan-at-Edly**

### CODE REVIEW

File: cms/envs/production.py [hunk 1]
```diff
@@ -537,7 +537,6 @@ def get_env_setting(setting):
 
 plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.CMS, plugin_constants.SettingsType.PRODUCTION)
 
-########################## Derive Any Derived Settings  #######################
+########################## Derive Any Derived Settings #######################
```
Comment (emzubair) [inline cms/envs/production.py:540]: remove this line

File: lms/envs/production.py [hunk 2]
```diff
@@ -934,6 +934,12 @@ def get_env_setting(setting):
 ########################## limiting dashboard courses ######################
 DASHBOARD_COURSE_LIMIT = ENV_TOKENS.get('DASHBOARD_COURSE_LIMIT', None)
 
+
+# ########################## Set Course Progress Reminder Email Days #######################
```
Comment (emzubair) [inline lms/envs/production.py:938]: initial `#` has is not part of formatting remove that

File: openedx/core/djangoapps/oauth_dispatch/jwt.py [hunk 3]
```diff
@@ -198,6 +198,9 @@ def _encode_and_sign(payload, use_asymmetric_key, secret):
     set_custom_metric('jwt_is_asymmetric', use_asymmetric_key)
     keys = jwk.KEYS()
 
+    # import pdb
```
Comment (emzubair) [inline openedx/core/djangoapps/oauth_dispatch/jwt.py:201]: remove commented code

**Review by emzubair — CHANGES_REQUESTED** [review]

File: cms/envs/production.py (hunk 1 above)
Comment (Faizan-at-Edly) [reply cms/envs/production.py:540]: Only a space was removed, this line is exactly the same.

**Pull request closed without merging**